tests/framework/PostProcessors/ETimporterPostProcessor/test_ETimporterMultipleET.xml

```diff
@@ -0,0 +1,56 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors/ETimporterPostProcessor</name>
+    <author>mandd</author>
+    <created>2017-11-07</created>
+    <classesTested>ETimporter</classesTested>
+    <description>
+      Tests of the ETimporter post-processor with multiple ETs
```

Comment:
more description

PS. classesTested should be ```Models.PostProcessors.ETImporter```

In addition, the name should be equal to how to run using the run_test command
In this case from 
```framework/PostProcessors/ETimporterPostProcessor```
it should be
```framework/PostProcessors.ETimporterPostProcessor```

since we can run this witht he following command

```./run_tests --re=framework/PostProcessors.ETimporterPostProcessor```